components/form-builder/translate/Description.tsx

```diff
@@ -18,12 +18,19 @@ export const Description = ({
 
   return (
     <>
-      <div className="text-entry">
-        <div className="section-heading">
+      <fieldset className="text-entry">
+        <legend className="section-heading">
           {t(element.type)}: {t("Description")}
-        </div>
+        </legend>
         <div className="section-text">
+          <label
+            className="sr-only"
+            htmlFor={`element-${element.id}-description-${translationLanguagePriority}`}
+          >
+            English text
```

Comment:
Should these be translated? 🤔

Reply:
done (also made them dynamic based on translationLanguagePriority)